.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -338,6 +338,11 @@ AddType text/vtt                            vtt
     # GitOps landing page
     RewriteRule ^container-platform/(4\.11|4\.12|4\.13|4\.14|4\.15|4\.16|4\.17|4\.18)/cicd/gitops/about-redhat-openshift-gitops.html /gitops/latest/understanding_openshift_gitops/about-redhat-openshift-gitops.html  [L,R=302]
 
+    # GitOps release notes version switch
+
+    RewriteRule ^gitops/1\.(8|9|10|11|12|13)/release_notes/gitops-release-notes-1-([0-9]+)\.html$ gitops/1.$1/release_notes/gitops-release-notes.html [L,R=302]
+    RewriteRule ^gitops/1\.(14|15|16)/release_notes/gitops-release-notes?(.*)\.html$ gitops/1.$1/release_notes/gitops-release-notes.html [L,R=302]
```

Comment:
I believe that this line will generate an infinite redirect loop.
https://docs.openshift.com/gitops/1.14/release_notes/gitops-release-notes-1-14.html will be redirected to
https://docs.openshift.com/gitops/1.14/release_notes/gitops-release-notes.html, which will be redirected to
https://docs.openshift.com/gitops/1.14/release_notes/gitops-release-notes.html, which will repeat until an error.

I would add the following line on line 344 to stop this loop:
`    RewriteRule ^gitops/1\.(14|15|16)/release_notes/gitops-release-notes.html$ - [L]`